config/application.rb

```diff
@@ -53,7 +53,7 @@ class Application < Rails::Application
       env: ENV.fetch('GOVUK_NOTIFY_ENV', 'development')
     ).symbolize_keys
 
-    config.x.support_email_address = 'apply-for-legal-aid@digital.justice.gov.uk'.freeze
+    config.x.support_email_address = 'abdirizak.obsiye@digital.justice.gov.uk'.freeze
```

Comment:
You should probably change this back, unless you want to receive all the support emails? 😁